### DISCUSSION TITLE
Is documentation available?

### DISCUSSION THREAD

**Discussion by znoooo** [imdhemy/appstore-iap#21] [Q&A]
Hello and really a nice work here! 
I can't find documentation about how to implement this library, there is something there? 

Thanks!

**Comment by imdhemy** [comment]
Just published a [getting started guide](https://imdhemy.github.io/laravel-iap-docs/), it's not complete at the moment, but I will update it ASAP.

**Comment by znoooo** [comment]
Oh thank you! Is missed the server notifications part (if is managed from library), do you have some suggestion? 
I've found something inside the code, but can't undestand how to use it
Thank you again
